can not create a new install

I tried to do a fresh android install but I'm struck at `make -C lib ../target/test/aar.stamp` which is failing with the following message :  
```shell
make: *** Aucune règle pour fabriquer la cible « ../target/test/aar.stamp ». Arrêt.`
```
which mean something like : No rules to build the target « ../target/test/aar.stamp ». Stopping.
If I try to revert to old DidKit version I have compilation errors like : 
```shell
error[E0061]: this function takes 3 arguments but 2 arguments were supplied
    --> lib/src/jni.rs:124:40
     |
124  |     let proof = rt.block_on(credential.generate_proof(&key, &options))?;
     |                                        ^^^^^^^^^^^^^^ ----  -------- supplied 2 arguments
     |                                        |
     |                                        expected 3 arguments
```

At the end of the installation process credible is "working" but there is no libdidkit.so library available. 
Same issue with stable or dev channel.

--- a/README.md
+++ b/README.md
@@ -178,7 +178,7 @@ and run:
 ```bash
 $ make -C lib install-rustup-android
 $ make -C lib ../target/test/java.stamp
-$ make -C lib ../target/test/aar.stamp
+$ make -C lib ../target/test/android.stamp
 $ make -C lib ../target/test/flutter.stamp
 $ cargo build
 ```

--- a/didkit_setup.sh
+++ b/didkit_setup.sh
@@ -72,7 +72,7 @@ cd ../didkit
 if [[ "$*" == *-android* ]]; then
   echo "Build didkit for Android"
   make -C lib install-rustup-android
-  make -C lib ../target/test/aar.stamp
+  make -C lib ../target/test/android.stamp
 fi
 
 if [[ "$*" == *-ios* ]]; then

